portfolio/src/main/webapp/jquery_functions.js

```diff
@@ -0,0 +1,31 @@
+/*
+* Cycles between  different background images, triggered by mouse click
+*/
+
+let img_arr = new Array(5);
+img_arr = ["../images/mountbierstadt.jpg", 
+           "../images/mountyale.jpg",
+           "../images/gardenofgods.jpg",
+           "../images/lakedillon.jpg",
+           "../images/mountelbert.jpg"
+           ];
+labels = ["the summit of Mount Bierdstadt.", 
+          "the summit of Mount Yale.", 
+          "the Garden of the Gods.", 
+          "Lake Dillon, seen from the vantage point of driving along Dillon Dam road.",
+          "the summit of Mount Elbert, the tallest point in Colorado."];
+
+
+$(document).ready(function(){
+    // if change background button is clicked, background image for the body is changed. 
+    // Appropriate image label is displayed.
+  let index = 0;
+  $("#change-background").click(function(){
+      index++;
```

Comment:
Do you ever get mount bierstadt to show up? It seems like the index can never be...

Reply:
From just playing around with the local server preview, Mt. Bierstadt does show up. I would like to add unit tests to ensure everything can show up. 
![bierstadt](https://user-images.githubusercontent.com/3937715/82944699-e8ce6400-9f58-11ea-8c77-8b7c2feaf7b1.png)